docs/core/deploying/trimming/prepare-libraries-for-trimming.md

```diff
@@ -64,6 +81,32 @@ To create your sample app, first create a separate application project with `dot
 </Project>
 ```
 
+:::zone-end
+
+:::zone pivot="dotnet-7-0"
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <PublishTrimmed>true</PublishTrimmed>
+    <!-- Prevent warnings from unused code in dependencies -->
+    <TrimMode>full</TrimMode>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="path/to/MyLibrary.csproj" />
+    <!-- Analyze the whole library, even if attributed with "IsTrimmable" -->
```

Comment:
```suggestion
    <!-- Analyze the whole library -->
```

No need to explain some interactions with other properties. Technically IsTrimmable has nothing to with this if TrimMode=full anyway.